thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeConfig.java

```diff
@@ -19,6 +22,17 @@
   private AnomalyMergeStrategy mergeStrategy = AnomalyMergeStrategy.FUNCTION;
   private long sequentialAllowedGap = 30_000; // 30 seconds
   private long maxMergeDurationLength = 12 * 60 * 60 * 1000; // 12 hours
+  private List<String> mergeablePropertyKeys = new ArrayList<>();
+
+  public AnomalyMergeConfig() {
+  }
+
+  public AnomalyMergeConfig(AnomalyMergeConfig other) {
```

Comment:
Good suggestion. @cecilynie, could you change this constructor to take variables instead of another config object?